cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -1582,6 +1582,24 @@ rules:
   - apiGroups: [""]
     resources: ["events"]
     verbs: ["list", "watch", "create", "update", "patch"]
+  - apiGroups: ["snapshot.storage.k8s.io"]
```

Comment:
These new rbac.need to be added for upgrade yaml file

Reply:
Yes, should be good in upgrade-from-v1.3-apply.yaml.
Note that it will not be applied for those who have already upgraded to v1.4.0. This should be ok if we just put a note in the release notes that this must be applied if you want to use the new cephfs snapshotting feature.

Reply:
Done. added to `1.3` upgrade yaml. Sure, after 1.4.0 they could apply the same and get it working.

Reply:
@travisn @Madhu-1 ptal.